src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -1,31 +1,66 @@
-import React, { useCallback, useMemo } from "react";
+import React, {
+  MouseEventHandler,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from "react";
 import classNames from "classnames";
 
-import { PosterPageVenue } from "types/venues";
+import { PersonalizedPoster, VenueEvent } from "types/venues";
 
 import { WithId } from "utils/id";
 
 import { PosterCategory } from "components/atoms/PosterCategory";
 
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBookmark as solidBookmark } from "@fortawesome/free-solid-svg-icons";
+import { faBookmark as regularBookmark } from "@fortawesome/free-regular-svg-icons";
+
+import { updatePersonalizedSchedule, savePosterToProfile } from "api/profile";
+import { useUser } from "hooks/useUser";
+
+// import { UserAvatar } from "components/atoms/UserAvatar";
+// import { useProfileModalControls } from "hooks/useProfileModalControls";
+// import { useWorldUsersById } from "hooks/users";
+
+import { useRelatedVenues } from "hooks/useRelatedVenues";
+import { useVenueEvents } from "hooks/events";
+
+import { WithVenueId } from "utils/id";
+
 import "./PosterPreview.scss";
 
 export interface PosterPreviewProps {
-  posterVenue: WithId<PosterPageVenue>;
   enterVenue: (venueId: string) => void;
+  personalizedPoster: WithId<PersonalizedPoster>;
 }
 
+const emptyRelatedEvents: WithVenueId<VenueEvent>[] = [];
+export const emptyPersonalizedSchedule = {};
+
 export const PosterPreview: React.FC<PosterPreviewProps> = ({
-  posterVenue,
   enterVenue,
+  personalizedPoster,
 }) => {
```

Comment:
As mentioned in other comments, this should be reverted to how it was previously.

Reply:
done